openedx/features/course_experience/tests/views/test_course_updates.py

```diff
@@ -61,14 +61,15 @@ def create_course_updates_block(course, user):
     return course_updates
 
 
-def remove_course_updates(course):
+def remove_course_updates(user, course):
     """
     Remove any course updates in the specified course.
     """
     updates_usage_key = get_course_info_usage_key(course, 'updates')
     try:
         course_updates = modulestore().get_item(updates_usage_key)
         course_updates.items = []
+        modulestore().update_item(course_updates, user.id)
```

Comment:
Question: only staff should be able to update course updates, I would think. Is this a staff user in all cases, or is it actually possible for learners to be able to publish updates themselves (that would be a problem!)?